root/lib/systemd/system/titus-imds-proxy@.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=Titus IMDS proxy for kubelet
+ConditionPathExists=/run/is_kubernetes
+
+StartLimitIntervalSec=30
+StartLimitBurst=10
+
+[Service]
+Type=notify
+NotifyAccess=all
+WatchdogSec=30
+
+Restart=on-failure
+KillMode=mixed
+
+User=www-data
```

Comment:
Why? Can we use https://www.freedesktop.org/software/systemd/man/systemd.exec.html#DynamicUser=

Reply:
I forgot to push some code here. This service runs as a privileged user on the host. If we feel strongly otherwise, we can muck with some mount locations to move to a more locked down user.